src/densities.jl

```diff
@@ -79,3 +79,23 @@ be one coefficient matrix per k-Point.
     count = sum(length(basis.ksymops[ik]) for ik in 1:length(basis.kpoints))
     from_fourier(basis, sum(ρaccus) / count)
 end
+
+"""
+Computes the *probability* (not charge) current, ∑ fn Im(ϕn* ∇ϕn)
+"""
+function compute_current(basis::PlaneWaveBasis, ψ::AbstractVector,
```

Comment:
I would rename it to `compute_probability_current` because it is a very odd use case and definitely not what I would expect from the name.

Reply:
Why do you explicitly type \psi and occupation?

Reply:
It's pretty clear no? It's the current of electrons (equal to minus the charge current). It's the one that's consistent with the electronic charge rho. I can call it electronic current if you want.
What do you mean by explicitly type?

Reply:
Why you make them `AbstractVector` instead of `Any`

Reply:
For the function: Maybe I misunderstood. Is this the generic expression for computing the electronic current density for bloch waves?

Reply:
Yes: https://en.wikipedia.org/wiki/Probability_current